### ISSUE TITLE
nodejs < 1:14.0

### ISSUE THREAD

**Issue by StackKorora** [OSC/ondemand#2261]
```
# cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.6 (Ootpa)

# dnf module list nodejs
Updating Subscription Management repositories.
Last metadata expiration check: 0:03:46 ago on Wed 24 Aug 2022 01:36:17 PM CDT.
Extra Packages for Enterprise Linux Modular 8 - x86_64
Name                                      Stream                                     Profiles                                                                 Summary                                              
nodejs                                    13                                         default, development, minimal                                            Javascript runtime                                   
nodejs                                    16-epel                                    default, development, minimal                                            Javascript runtime                                   

Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
Name                                      Stream                                     Profiles                                                                 Summary                                              
nodejs                                    10 [d]                                     common [d], development, minimal, s2i                                    Javascript runtime                                   
nodejs                                    12                                         common [d], development, minimal, s2i                                    Javascript runtime                                   
nodejs                                    14 [e]                                     common [d], development, minimal, s2i                                    Javascript runtime                                   
nodejs                                    16                                         common [d], development, minimal, s2i                                    Javascript runtime                                   

Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled

# dnf install ondemand
Updating Subscription Management repositories.
Last metadata expiration check: 0:03:47 ago on Wed 24 Aug 2022 01:36:17 PM CDT.
Error: 
 Problem: package ondemand-nodejs-2.0-2.el8.x86_64 requires nodejs < 1:14.0, but none of the providers can be installed
  - cannot install the best candidate for the job
  <snip excess>
```

  Doesn't look like the RPMs for RHEL8 have the #1382 update. Confirmed that if I downgrade nodejs to 1.12 that OOD installs.
  Thanks!

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1202857710727819) by [Unito](https://www.unito.io)

**Comment by treydock** [comment]
OnDemand 2.0 only supports NodeJS 12.  You cannot have the nodejs:14 module enabled with OnDemand 2.0 releases.

**Comment by treydock** [comment]
I see where the confusion lies, our documentation refresh accidentally pulled in wrong nodejs when we merged some changes to make 2.0 docs behave like 2.1 (unreleased) docs.  Fix is here: https://github.com/OSC/ood-documentation/pull/719

**Comment by treydock** [comment]
Documentation has been corrected: https://osc.github.io/ood-documentation/latest/installation/install-software.html
